modules/apps/mentions/mentions-web/src/main/java/com/liferay/mentions/web/internal/editor/configuration/MBMentionsEditorConfigContributor.java

```diff
@@ -26,7 +26,7 @@
 	property = {
 		"editor.config.key=bodyEditor", "editor.config.key=replyMBEditor",
 		"editor.name=alloyeditor", "editor.name=alloyeditor_bbcode",
-		"editor.name=ckeditor", "editor.name=ckeditor_bbcode",
+		"editor.name=ckeditor_bbcode", "editor.name=ckeditor_classic",
```

Comment:
Hi @marcogalluzzi 

Can you check if this happens too on Blogs?

thanks!

Reply:
Yes, I'll do it

Reply:
<img src="https://media3.giphy.com/media/9Gp5ZwY8FRvna/giphy.gif"/>

Reply:
We don't know if anyone was using the `ckeditor_classic` in Blogs, however I've added the autocompletion feature to this editor just in case